tests/integration/ovs_test.py

```diff
@@ -163,6 +163,9 @@ def test_ovs_interface_with_max_length_name():
     assertlib.assert_absent(ovs_interface_name)
 
 
+@pytest.mark.xfail(
```

Comment:
This can be a `skip`, just a nit. Thanks!

Reply:
This is a bug.
`skip` means we accept the current state (due to the env usually) and there is no plan to "fix" it.
`xfail` means we intend to fix it and remove it.

The difference is very thin, I like to think of it as: `xfail` means that we intend to treat it.

Reply:
Oh, thanks for explaining. LGTM